### ISSUE TITLE
fifo_empty

### ISSUE THREAD

**Issue by soodoju** [2023-Spring-ECE4278/lab4#1]
DMAC_1.2 SPEC상 fifo_empty case는 생기지 않을 것 같은데 코드상 반영은 필요할까요?
반영 시 생기지 않는 case이고 만약 발생한다면 deadlock이 유발될 것 같아 질문드립니다.

**Comment by Park-So-Young** [comment]
네, 말씀 해주신대로 DMAC specification v1.2의 FSM에 따라 설계하면 fifo_empty가 발생하지 않습니다.
Lab 실습에서는 최적화보다는 SPEC에 따라 동작하도록 설계하고 검증하는 것을 우선 목적으로 하고 있어 
모듈 간에 사용하지 않는 input, output도 전부 wiring해 놓았기 때문에, 꼭 모든 신호를 사용하지 않으셔도 됩니다.
추후 프로젝트에서는 이러한 경우 면적을 줄이기 위해 module instantiation시에 wiring하지 않을 수 있을 것 같습니다.
